packages/vscode-extension/src/envTree.ts

```diff
@@ -117,3 +119,43 @@ async function localSettingsExists(projectRoot: string): Promise<boolean> {
   const provider = new LocalSettingsProvider(projectRoot);
   return await fs.pathExists(provider.localSettingsFilePath);
 }
+
+export async function addSubscriptionAndResourceGroupNode(env: string) {
+  if (!environmentTreeProvider) {
+    return;
+  }
+
+  const parentCommand = environmentTreeProvider.findCommand("fx-extension.environment." + env);
```

Comment:
Shall we return directly if the env value equal to local?

Reply:
Sounds good as we already include 'local' env node in the tree view now.